lib/classes/Swift/Signers/OpenDKIMSigner.php

```diff
@@ -53,14 +53,10 @@ public function addSignature(Swift_Mime_SimpleHeaderSet $headers)
 
     public function setHeaders(Swift_Mime_SimpleHeaderSet $headers)
     {
-        $bodyLen = $this->bodyLen;
-        if (is_bool($bodyLen)) {
-            $bodyLen = -1;
-        }
         $hash = 'rsa-sha1' == $this->hashAlgorithm ? OpenDKIMSign::ALG_RSASHA1 : OpenDKIMSign::ALG_RSASHA256;
         $bodyCanon = 'simple' == $this->bodyCanon ? OpenDKIMSign::CANON_SIMPLE : OpenDKIMSign::CANON_RELAXED;
         $headerCanon = 'simple' == $this->headerCanon ? OpenDKIMSign::CANON_SIMPLE : OpenDKIMSign::CANON_RELAXED;
-        $this->dkimHandler = new OpenDKIMSign($this->privateKey, $this->selector, $this->domainName, $headerCanon, $bodyCanon, $hash, $bodyLen);
+        $this->dkimHandler = new OpenDKIMSign($this->privateKey, $this->selector, $this->domainName, $headerCanon, $bodyCanon, $hash, -1);
```

Comment:
-1 means “sign it all”. We want that. `$this->bodyLen` has not been initialised at this point.